src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -21,27 +21,39 @@
 
 
 # export users
-export_data.to_csv(request_client.get_users(), f"{schoology_output_path}/users.csv")
+users_response = request_client.get_users()
+users_list = []
+while True:
+    users_list = users_list + users_response.current_page_items
+    if users_response.get_next_page() is None:
```

Comment:
users_response.get_next_page() returns self when there is another page, and returns None when is the last page.